app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerExecutionDetails.html

```diff
@@ -0,0 +1,60 @@
+<div ng-controller="createLoadBalancerExecutionDetailsCtrl">
+    <execution-details-section-nav sections="configSections"></execution-details-section-nav>
+    <div class="step-section-details" ng-if="detailsSection === 'loadBalancerConfig'">
+        <div class="row">
+
+            <table class="table table-condensed">
+                <thead>
+                    <tr>
+                        <th>Account</th>
+                        <th>Name</th>
+                        <th>Region</th>
+                        <th ng-if="provider === 'aws'">Subnet</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr ng-repeat="loadBalancer in stage.context.loadBalancers">
+                        <td>
+                            <account-tag account="loadBalancer.credentials"></account-tag>
+                        </td>
+                        <td>
+                            {{ loadBalancer.name }}
+                        </td>
+                        <td>
+                            {{ loadBalancer.region }}
+                        </td>
+                        <td ng-if="provider === 'aws'">
+                            {{ loadBalancer.subnetType || '[none]' }}
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <stage-failure-message is-failed="stage.isFailed" message="stage.failureMessage"></stage-failure-message>
+
+        <div class="row" ng-if="createdLoadBalancers.length">
+            <div class="col-md-12">
+                <div class="well alert alert-info">
+                    <strong>Created:</strong>
+                    <span ng-repeat="lb in createdLoadBalancers">
+                        <a target=_blank href="http://{{lb.dnsName}}">
+                            </br>{{lb.dnsName}}
+                        </a>
+                        <copy-to-clipboard
+                                class="copy-to-clipboard copy-to-clipboard-sm"
+                                text="http://{{lb.dnsName}}"
+                                tool-tip="Copy DNS Name to clipboard">
+                        </copy-to-clipboard>
+                    </span>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="step-section-details" ng-if="detailsSection === 'taskStatus'">
+        <div class="row">
+            <execution-step-details item="stage"></execution-step-details>
+        </div>
+    </div>
+</div>
```

Comment:
Two spaces is preferred to 4 spaces. There are probably some files using 4 spaces scattered throughout the application, but we're trying to stick with 2 on new code.